File: catorce ejercicios/PYTHON2/ejemplo.3.py
#Diseñe un algoritmo que dado dos números A y B, determine si A es mayor, igual o menor a B.
print ("Ejemplo 3")

numero_a = int(input("Ingrese el valor de A: "))
numero_b = int(input("Ingrese el valor de B: "))

def comparar_numeros(a, b):
    if a > b:
        return "A es mayor que B"
    elif a < b:
        return "A es menor que B"
    else:
        return "A es igual a B"
        
resultado = comparar_numeros(numero_a, numero_b)
print(resultado)

File: catorce ejercicios/JAVASCRITP2/Javascripts/ejemplo.3.js
/* Diseñe un algoritmo que dado dos números A y B, determine si A es mayor, igual o menor a B. */

function compararNumeros() {
    let numeroA = parseFloat(document.getElementById("numeroA").value);
    let numeroB = parseFloat(document.getElementById("numeroB").value);

    let resultado = document.getElementById("resultado");

    if (isNaN(numeroA) || isNaN(numeroB)) {
        resultado.innerHTML = "Error: Ingrese números válidos en ambos campos.";
    } else {
        if (numeroA === numeroB) {
            resultado.innerHTML = " A es igual a  B.";
        } else if (numeroA < numeroB) {
            resultado.innerHTML = " A es menor que B.";
        } else {
            resultado.innerHTML = " A es mayor que B.";
        }
    }
}

